katas/crashOverride.js

```diff
@@ -0,0 +1,68 @@
+(firstName = {
```

Comment:
You could also define firstName and surname as constants?

Reply:
sorted?